librarywidget.py

```diff
@@ -525,21 +524,22 @@ def foldersWidget(self):
         """
         return self._foldersWidget
 
-    def folderIgnoreFilter(self):
+    def folderIgnoreFilter(self, path):
```

Comment:
What do you think of changing the name to "setFilter" instead of "setIgnoreFilter" and the new filter method would be "isFolderValid"?

Example:

```python
class LibraryWidget(QWidget):

    def setRootPath(self):
        ...
        self.foldersWidget().setFilter(self.isFolderValid)

    def isFolderValid(path):
        if os.path.isdir(path):
            return True
        else:
            return False
```

Reply:
Yes, `Ignore` and `Filter` is quite same semantics, but `isFolderValid` loses filtering semantics at all. If this is about to test if folder is valid for LibraryWidget it is ok, so we will use `lambda x: not self.isFolderValid(x)` in `setFilter`.

Reply:
those names are not so bad i got it when i had attempt to rename. Having simple `setFilter` will override `QFileSystemModel.setFilter` method.